easybuild/easyblocks/q/quantumespresso.py

```diff
@@ -338,16 +447,21 @@ def sanity_check_step(self):
                         "virtual.x"]
             if LooseVersion(self.version) > LooseVersion("5"):
                 upftools.extend(["interpolate.x", "upf2casino.x"])
+            if LooseVersion(self.version) >= LooseVersion("6.3"):
+                upftools.extend(["fix_upf.x"])
 
         if 'vdw' in self.cfg['buildopts']:  # only for v4.x, not in v5.0 anymore
             bins.extend(["vdw.x"])
 
         if 'w90' in self.cfg['buildopts']:
-            bins.extend(["wannier90.x"])
+            if LooseVersion(self.version) < LooseVersion("6"):
+                bins.extend(["w90chk2chk.x"])
```

Comment:
@akesandgren Version checking doesn't match what's in `install_step`, making the sanity check fail for QE 4.2 for example...

Reply:
Doh.... will fix.